lib/std/simd.zig

```diff
@@ -278,40 +278,113 @@ test "vector shifting" {
     try std.testing.expectEqual([4]u32{ 40, 30, 20, 10 }, reverseOrder(base));
 }
 
+/// Return the index of the first true bool in `vec` or null if there
+/// is none.
 pub fn firstTrue(vec: anytype) ?VectorIndex(@TypeOf(vec)) {
     const len = vectorLength(@TypeOf(vec));
-    const IndexInt = VectorIndex(@TypeOf(vec));
 
-    if (!@reduce(.Or, vec)) {
-        return null;
+    // FIXME: add other backends when they implement bit-casting the same way
```

Comment:
No TODO comments please (#363). File an issue if there is follow-up work to be done.